content/en/09-release-notes/75-3.0.md

```diff
@@ -0,0 +1,84 @@
+- **Frequent DB writes** no longer create excessive number of commits – [#512](https://github.com/versionpress/versionpress/issues/512). Things like view counters, Akismet spam count etc. are now committed once in an hour (configurable in the future) or before actions like cloning and merging. This avoiding floods of not very useful commits.
+- **Configurable entity ignoring** and, specifically, **better spam comment handling** – [#511](https://github.com/versionpress/versionpress/issues/511). Some entities just should not be part of the repository, e.g., transient options or spam comments. Previously, some of this was hardcoded (e.g., transient options) or not handled very well at all (e.g., spam comments). VersionPress 3.0 introduces a better, extensible ignoring mechanism that will be user-configurable in the future.
+- **Ignoring of specific database columns** – [#902](https://github.com/versionpress/versionpress/issues/902). Similarly to ignoring entities (database rows), VersionPress can also ignore database columns. These columns are either plain ignored (for example, VersionPress doesn't care about `option_id`, only `option_name`) or ignored with a reference to a PHP function that computes the values (for example, post's `comment_count` is computed dynamically based on the actual comments).
+- **Initial support for custom directory layouts** – [#565](https://github.com/versionpress/versionpress/issues/565). WordPress location, `wp-content`, `plugins` or `uploads` directories can now all be customized and VersionPress will support that. The support is initial, for example, the structure needs to be defined before VersionPress is activated, more complex setups like Bedrock are not fully supported yet, etc., but it's a good first step.
+- **Data updates via `query()` are now understood** – [#510](https://github.com/versionpress/versionpress/issues/510). Previously, VersionPress handled data modified via `insert()`, `update()` or `delete()` methods. These are quite simple to support and WordPress core use them 99 % of the time so we started with that, but VersionPress 3.0 is much more reliable as it also supports the `query()` method and can now understand 100 % of the data updates.
+- **Common (shared) config** – [#630](https://github.com/versionpress/versionpress/issues/630). Most things in `wp-config.php` are environment-specific, for example, database connection, so VersionPress gitignored this file since 1.0-beta2. However, some config values should be shared, for example, custom folder constants `WP_CONTENT_DIR` or `WP_PLUGIN_DIR`. For this, VersionPress 3.0 adds a config file called `wp-config.common.php` which is committed into the repository and `require`'d from the standard `wp-config.php` file. We also ship a `wp vp config` WP-CLI command to manipulate the configuration constants.
```

Comment:
`wp vp config` supports only public VP constants. This may sound like it can configure all constants.